### ISSUE TITLE
User can't select cards on dashboard (views) via keyboard

### ISSUE THREAD

**Issue by ssciolla** [tl-its-umich-edu/my-learning-analytics#1473]
Thank you for contributing to this project!

- Make sure to search the [issues](https://github.com/tl-its-umich-edu/my-learning-analytics/issues) for duplicates first!

### Expected behavior (A description of what you expected to happen) : 

The list of views should be keyboard accessible, i.e. you should be able to use Tab and Shift + Tab to navigate through them and back, and press Enter to select one.

### Describe the bug (Tell us what happens instead of the expected behavior) :

The user can Tab through the list of view cards, but pressing Enter when the view cards is in focus does not send the user to the corresponding view.

### Additional context (Add any other context about the problem here) :

Looking at the source code, it looks like this was partially the result of an attempt to include the help link in the card. Nested `a` tags are invalid HTML I guess, so the solution at the time was to separately link different parts of the card (title, description, image. etc.) but give them all `tabindex=-1`. The card itself still has `tabindex=0` but there is no link attached to it.

Some initial research suggests that one solution may to use CSS to layer the help link on top of card. See https://css-tricks.com/nested-links/

**Comment by jennlove-um** [comment]
Testing passes in beta.
